modules/wmco-prerequisites.adoc

```diff
@@ -7,78 +7,52 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
-== Supported platforms based on {product-title} and WMCO versions
+[id="wmco-prerequisites-supported-5.1.0_{context}"]
+== WMCO 5.1.0 supported platforms and Windows Server versions
 
-[cols="5",options="header"]
-|===
-|Platform
-|Supported {product-title} version
-|Supported WMCO version
-|Installer-provisioned infrastructure installation support
-|User-provisioned infrastructure installation support
-
-|Amazon Web Services (AWS)
-|4.6+
-|WMCO 1.0+
-|GA
-|
+////
+The Windows Machine Config Operator {WMCO) 5.1.0 supports two methods for deploying Windows worker nodes: 
```

Comment:
In the Operator Hub [description](https://github.com/openshift/windows-machine-config-operator/blob/master/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml#L25), it is stated like this:
> Windows instances can be added either by creating a [MachineSet](https://docs.openshift.com/container-platform/latest/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws),
    or by specifying existing instances through a [ConfigMap](https://docs.openshift.com/container-platform/latest/windows_containers/configuring-byoh-windows-instance.html).

Reply:
I wonder if this info should go elsewhere. It's not really a prerequisite, right? Maybe in here: [Red Hat OpenShift support for Windows Containers overview](http://file.rdu.redhat.com/mburke/winc-release-notes-tables-5.0/windows_containers/)? Second paragraph and NOTE